### DISCUSSION TITLE
Error:The domain *my domain* is already owned by another Vercel account.

### DISCUSSION THREAD

**Discussion by Jhonlozanot** [vercel/community#423] [Help]
Dear, how can I solve this, the domain that I want to add is my property and I cannot add it to my panel

**Comment by Pieparker** [comment]
Hey there! For us to help, can you share the domain name and your team name/slug, please?

It's likely that your domain is in use on another Vercel account, either by:
- the person who previously owned the domain name
- another one of your accounts
- another service (Hashnode or Super.so, for example, like was the case for [this person](https://www.essaidi.me/Domain-is-already-owned-by-another-Vercel-account-Solution)) who uses Vercel
